.github/workflows/run-metrics.yaml

```diff
@@ -40,3 +40,13 @@ jobs:
       - name: run memory usage inside container test
         run:  bash tests/metrics/gha-run.sh run-test-memory-usage-inside-container
 
+      - name: Make metrics tarball ${{ matrix.vmm }}
```

Comment:
should we use lowercase in all the names to have uniformity?

Reply:
@GabyCT change applied, thanks